jobbergate-cli/jobbergate_cli/subapps/applications/app.py

```diff
@@ -405,3 +405,89 @@ def download_files(
         ).strip(),
         subject="Application download succeeded",
     )
+
+
+@app.command()
+@handle_abort
+def clone(
+    ctx: typer.Context,
+    id: Optional[int] = typer.Option(
+        None,
+        help=f"The specific id of the application. {ID_NOTE}",
+    ),
+    identifier: Optional[str] = typer.Option(
+        None,
+        help=f"The human-friendly identifier of the application. {IDENTIFIER_NOTE}",
+    ),
+    application_identifier: Optional[str] = typer.Option(
+        None,
+        help="Optional new application identifier to override the original",
```

Comment:
I think this text should also explain that the new identifier cannot match the old one.

Reply:
Solved in [110b589](https://github.com/omnivector-solutions/jobbergate/pull/460/commits/110b58961ea8c1694ef26da8476ef5ad6ace3784).